api/config/config.exs

```diff
@@ -11,4 +11,7 @@ config :openvac, OpenVacWeb.Endpoint,
 config :openvac,
   ecto_repos: [OpenVac.Repo]
 
-config :openvac, OpenVac.Repo, url: "ecto://0.0.0.0:5432/openvac?sslmode=disable"
+config :openvac, OpenVac.Repo,
+  url: "ecto://0.0.0.0:5432/openvac?sslmode=disable",
+  username: "postgres",
```

Comment:
Creds are only need if you're working in a dockerized postgres pa.

Reply:
oh yeah. let me re move for now kruch.